test/unit/renderer/viz/expressions/aggregation/clusterAggregation.test.js

```diff
@@ -60,5 +63,18 @@ describe('src/renderer/viz/expressions/clusterAggregation', () => {
             const actual = s.clusterSum($price).eval(fakeFeature);
             expect(actual).toEqual(fakeFeature._cdb_agg_sum_price);
         });
+
+        it('clusterCount() should return fakeFeature._cdb_feature_count if defined', () => {
+            const actual = s.clusterCount().eval(fakeFeature);
+            expect(actual).toEqual(fakeFeature._cdb_feature_count);
+        });
+
+        it('clusterCount() should return 1 if _cdb_feature_count is undefined in the feature', () => {
+            const aFeature = Object.assign({}, fakeFeature);
+            delete aFeature._cdb_feature_count;
+
+            const actual = s.clusterCount().eval(aFeature);
```

Comment:
Just passing `{}` should be ok and then you could remove the construction of `aFeature`

Reply:
Nice simplification!